### PR TITLE
Improvement: Add Incoming Tablist Event to Custom Scoreboard

### PR DESCRIPTION
**Pull request by j10a1n15**
## What
This Pull Request adds the option to also show the incoming tablist event to the Custom Scoreboard. I do not want to auto add this to everyones Events Dropdown.

## Changelog Improvements
+ Added incoming tab list event to Custom Scoreboard. - j10a1n15

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/features/gui/customscoreboard/ScoreboardEvents.kt [hunk 1]
```diff
@@ -158,7 +158,12 @@ enum class ScoreboardEvents(
     ACTIVE_TABLIST_EVENTS(
         ::getActiveEventLine,
         ::getActiveEventShowWhen,
-        "§7(All Active Tablist Events)"
+        "§dHoppity's Hunt\n §fEnds in: §e26h"
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/features/gui/customscoreboard/ScoreboardEvents.kt]: this looks wrong

**Review by hannibal002 — CHANGES_REQUESTED** [review]

File: src/main/java/at/hannibal2/skyhanni/features/gui/customscoreboard/ScoreboardEvents.kt (hunk 1 above)
Comment (j10a1n15) [reply src/main/java/at/hannibal2/skyhanni/features/gui/customscoreboard/ScoreboardEvents.kt]: explain

Comment (hannibal002) [reply src/main/java/at/hannibal2/skyhanni/features/gui/customscoreboard/ScoreboardEvents.kt]: is this still the "active tablist events" thing? why is there the hoppity hunt text as template?

Comment (j10a1n15) [reply src/main/java/at/hannibal2/skyhanni/features/gui/customscoreboard/ScoreboardEvents.kt]: It is yes, what is your other idea

**Review by hannibal002 — APPROVED** [review]

**Pull request merged**